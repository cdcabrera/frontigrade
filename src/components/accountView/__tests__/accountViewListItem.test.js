import React from 'react';
import { mount } from 'enzyme';
import AccountViewListItem from '../accountViewListItem';

describe('AccountViewListItem Component', () => {
  it('should render', () => {
    const props = {
      item: {
        arn: 'arn:aws:iam::TEST',
        creation_date: '2018-07-07',
        id: '1',
        images: 1,
        instances: 2,
        name: 'Lorem test',
        openshift_instances: null,
        rhel_instances: 1,
        type: 'aws',
        user_id: 1
      }
    };

    const component = mount(<AccountViewListItem {...props} />);

    expect(component.render()).toMatchSnapshot();
  });
});
