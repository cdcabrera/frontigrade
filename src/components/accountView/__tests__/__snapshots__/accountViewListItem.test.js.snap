// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccountViewListItem Component should render 1`] = `
<div
  class="list-group-item"
>
  <div
    class="list-view-pf-actions"
  >
    <div />
  </div>
  <div
    class="list-view-pf-main-info"
  >
    <div
      class="list-view-pf-body"
    >
      <div
        class="list-view-pf-description"
      >
        <div
          class="list-group-item-text"
        >
          <div
            class="list-group-item-heading"
          >
            Lorem test
          </div>
          <div
            class="list-group-item-text"
          >
            Created 12:00AM, July 7th 2018
          </div>
        </div>
      </div>
      <div
        class="list-view-pf-additional-info"
      >
        <div
          class="list-view-pf-additional-info-item cloudmeter-listview-infoitem"
        >
          <span>
            <span
              aria-hidden="true"
              class="pficon pficon-cluster"
            />
            <strong>
              1
            </strong>
             Images
          </span>
        </div>
        <div
          class="list-view-pf-additional-info-item cloudmeter-listview-infoitem"
        >
          <span>
            <span
              aria-hidden="true"
              class="pficon pficon-screen"
            />
            <strong>
              2
            </strong>
             Instances
          </span>
        </div>
        <div
          class="list-view-pf-additional-info-item cloudmeter-listview-label cloudmeter-listview-label-has-badge"
        >
          <span>
            <strong>
              1
            </strong>
            <span
              class="label label-warning"
            >
              <abbr
                title="Red Hat Enterprise Linux"
              >
                RHEL
              </abbr>
            </span>
          </span>
        </div>
        <div
          class="list-view-pf-additional-info-item cloudmeter-listview-label cloudmeter-listview-label-has-badge"
        >
          <span>
            <strong>
              N/A
            </strong>
            <span
              class="label label-primary"
            >
              <abbr
                title="Red Hat OpenShift Container Platform"
              >
                RHOCP
              </abbr>
            </span>
          </span>
        </div>
      </div>
    </div>
  </div>
</div>
`;
